java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -127,10 +173,26 @@ public void subscribe() throws URISyntaxException, IOException, InterruptedExcep
         }
     }
 
+    @SuppressWarnings({"Regexp"})
```

Comment:
Which error are you getting for needing this?

Reply:
`Line matches the illegal pattern '\.to(Lower|Upper)Case\(\)'. [Regexp]`

Reply:
Maybe following the next comment we can get rid of the entire method.